lib/MetaCPAN/Client/Request.pm

```diff
@@ -213,6 +214,16 @@ sub _read_filters {
     return ( filter => $filter );
 }
 
+sub _read_sort {
+    my $self   = shift;
+    my $params = shift;
+
+    my $sort = delete $params->{sort};
+    ref($sort) or return ();
```

Comment:
If we're already using `Ref::Util`, might as well use `is_ref` here too.

Reply:
sure, that's a c/p of older code since before the move to Ref::Util.
fixed on master.